web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -2328,17 +2312,6 @@
                   }),
                   title: title || "Bing Aerial"
                 });
-              case "stamen":
-                //We make watercolor the default layer
-                var type = opt && opt.name ? opt.name : "watercolor",
-                  source = new ol.source.Stamen({
-                    layer: type
-                  });
-                source.set("type", type);
-                return new ol.layer.Tile({
-                  source: source,
-                  title: title || "Stamen"
-                });
```

Comment:
This switch/case does not appear to have a default...

If we are removing, rather than offering *stadiamaps* alternative we should default to *osm* or something that will display rather than being empty?

Reply:
OSM is the default AFAIK